### DISCUSSION TITLE
GO on SuSE Tumbleweed - problem after latest updates

### DISCUSSION THREAD

**Discussion by mayesp** [GrandOrgue/grandorgue#1339] [Help]
I am running GrandOrgue on SuSE Tumbleweed. I tend to do a "zypper dup" almost every week to pick up the latest fixes.

Recently this failed, as it appears that the repository I was using, which I think was called  home:/e9925248:/grandorgue (from looking in zypper.log) has disappeared. And there was a conflict between the installed version from that repository and version 3.9.4 which now seems to be in the main SuSE repository.

I don't remember the full details, but I think that is what happened - should have taken notes!

Anyway, I uninstalled the old version, and now have grandorgue-3.9.4-1.1 installed. When I first launched it, I only had access to the demo organ. I saw that there was a new file GrandOrgueConfig in my home directory. I found a file of the same name in $HOME/.config. I moved the new one aside, and copied the one from .config to my home directory. Things are better - the organs I used are now there - mostly I use Bureå (extended AGO) - and all the piston settings are there when I use the mouse to select them.

But none of the keyboards are recognised, nor is the Novation Launchpad I use.

I *think* this is down to the .cmb files. The most recent one I have from before the upgrade is in $HOME/.config/GrandOrgueData. And I have one created yesterday when I first fired up the new GO in $HOME/Documents/GrandOrgue/Data. But they have completely different 40-hex-digit filenames.

Obviously I could just re-detect my keyboards, pedalboard and Launchpad, but I would like to recover what I had before, which has been working for several years.

How can I use my old .cmb file, assuming this is the right thing to do? And is the location of these configuration files documented? They seem to have changed since I first started using GO.

Thanks in advance -- Peter

**Comment by kerkovits** [comment]
The location of the configuration files is now customizable in the Paths panel of the Files->Settings dialog.

The last version of the old repository was 0.3.1.2330. An RPM build of this version for Fedora still exists at https://download.copr.fedorainfracloud.org/results/ycollet/audinux/fedora-37-x86_64/04839668-GrandOrgue/ You may try if this binary works with OpenSUSE. If it works, you could read the old cmb files with it, do File->Export settings and combinations for each organ. With this you get human-readable file names. You could import that into the new version of GO via File->Import settings and Import combinations.

If running the old version is unsuccessful, you may still try to import cmb files from the old version, provided that you can identify which cmb is for which organ.
